channeldb/options.go

```diff
@@ -40,9 +42,13 @@ type Options struct {
 // DefaultOptions returns an Options populated with default values.
 func DefaultOptions() Options {
 	return Options{
+		BoltBackendConfig: kvdb.BoltBackendConfig{
```

Comment:
small nit: the dbpath and dbfilename aren't populated here, which could be confusing. i do see why we would do this tho since the fields just get copied over, i'm okay leaving it like this

Reply:
I didn't want to create those duplicate structs, so I ended up embedding `kbdb.BoltBackendConfig` into the `Options` struct, even though in this context the db path and filename aren't really used. I think we'll clean this up once we deprecate the old `channeldb.Open()` function.